libpod/pod_api.go

```diff
@@ -449,12 +479,18 @@ func (p *Pod) Status() (map[string]define.ContainerStatus, error) {
 	if !p.valid {
 		return nil, define.ErrPodRemoved
 	}
-
+	noInitCtrs := make([]*Container, 0)
```

Comment:
Should do `make([]*Container, 0, len(allCtrs))` (and do this a few lines down so allCtrs exists)

Reply:
> Should do `make([]*Container, 0, len(allCtrs))` (and do this a few lines down so allCtrs exists)

I don't believe this can be done because not allCtrs go into InitCtrs.